examples/intrinsics.rs

```diff
@@ -5,6 +5,7 @@
 
 #![allow(unused_features)]
 #![allow(stable_features)] // bench_black_box feature is stable, leaving for backcompat
+#![allow(internal_features)]
```

Comment:
Btw, bootstrap compiler won't say anything about unknown lint?

Reply:
I believe the bootstrap compiler allows warnings, but this may have changed since I last checked.

Reply:
Builds with v0.1.100.